docs/zh_cn/3_exist_data_new_model.md

```diff
@@ -277,7 +284,7 @@ python tools/train.py configs/cityscapes/cascade_mask_rcnn_r50_augfpn_autoaug_10
 为了能够测试训练好的模型，你可以运行如下命令：
 
 ```shell
-python tools/test.py configs/cityscapes/cascade_mask_rcnn_r50_augfpn_autoaug_10e_cityscapes.py work_dirs/cascade_mask_rcnn_r50_augfpn_autoaug_10e_cityscapes.py/latest.pth --eval bbox segm
+python tools/test.py configs/cityscapes/cascade-mask-rcnn_r50_augfpn_autoaug-10e_cityscapes.py work_dirs/cascade-mask-rcnn_r50_augfpn_autoaug-10e_cityscapes/epoch_10.pth
```

Comment:
why we use epoch_10

Reply:
latest.pth is not supported in mmengine for now, and last_checkpoint is only for resume_from not load_from